target/linux/pistachio/modules.mk

```diff
@@ -192,48 +192,3 @@ define KernelPackage/nhc_fragment/description
 endef
 
 $(eval $(call KernelPackage,nhc_udp))
```

Comment:
We will not need anymore the definition in modules.mk once the wifi driver is out of the kernel since it will be in Ci40-platform-feed.
I should remove kmod-uccp420wlan and kmod-cfg80211 from target.mk for now.
The patch for mac80211 could be removed but it avoids doing a lot of changes in the wifi driver, that is why I preferred modifying mac80211.